Integrations/integration-Tanium.yml

```diff
@@ -393,16 +424,40 @@ script:
         if response.get('question_results'):
             export_kwargs = {}
             export_kwargs['obj'] = response['question_results']
-            export_kwargs['export_format'] = 'csv'
+            export_kwargs['export_format'] = 'xml'
+            export_kwargs['header_sort']=False
             out = handler.export_obj(**export_kwargs)
 
-            result = csvstr_to_list(out)
 
-            ec = { 'Tanium.QuestionResults' : result }
-            return create_entry(
-                    'Result for parsed query - %s' % (query_text,),
+            #tanium built in csv output is broken for Multiple columns with the same name.
+            csv_string = parse_xml_to_csv(out)
+            result = csvstr_to_list(csv_string)
+            flat_length = len(list(chain.from_iterable(result)))
+            ttlrecords = len(result)
+            query_text = query_text.replace(r"[",r"\[").replace(r"]","\]")
+            MAX_ENTRIES=1000
+            if flat_length > MAX_ENTRIES:
```

Comment:
Not sure why do we need this part. The old way we did things did not work?

Reply:
If the context entry is too big it will NOT be entered into context at all.  I needed to set a limit of results.  I found this out by doing a search for the hash of a blank file in our environment.  Tanium returned multiple files for thousands of machines.  The results never were injected into context because it was too big.

Reply:
@shaniacht1 is it a bug on our end?